.gitignore

```diff
@@ -52,3 +52,6 @@ realm/realm-library/src/main/cpp/jni_include
 realm/realm-library/distribution
 # Cmake output
 realm/realm-library/.externalNativeBuild
+
+gen/
```

Comment:
Where do these come from?

Reply:
Last time I saw `gen` it came from Eclipse things. 😮

Reply:
@cmelchior @Zhuinden  I am not sure from where It came, I have imported the whole project in IntelliJ Ultimate, and i guess the build generated the gen folder so I have added it in the .gitignore, to ignore then gen folder.